draft-ietf-rats-corim.md

```diff
@@ -235,21 +235,20 @@ The CDDL definitions in this document follows the naming conventions illustrated
 
 A CoRIM is a collection of tags and related metadata in a concise CBOR {{-cbor}} encoding.
 A CoRIM can be digitally signed with a COSE {{-cose}} signature.
-A tag identifies and describes properties of modules or components of a system.
+A tag is a structured, machine-readable metadata format used to uniquely identify, describe and manage modules or components of a system.
 
 Tags can be of different types:
 
 * Concise Module ID (CoMID) tags ({{sec-comid}}) contain metadata and claims about the hardware and firmware modules.
 
-* Concise Software ID (CoSWID) tags ({{-coswid}}) describe software components.
+* Concise Software ID (CoSWID) tags ({{-coswid}}) uniquely identify, describe, and manage software components.
 
 * Concise Tag List (CoTL) tags ({{sec-cotl}}) contain the list of CoMID and CoSWID tags that the Verifier should consider as "active" at a certain point in time.
 
-The set of tags is extensible so that future specifications can add new kinds of information.
+The collection of tags is extensible so that future specifications can add new types of tags.
```

Comment:
This is inaccurate. There is an extension socket that allows other kinds of information to be added. The tag list's use of `tag-identity-map` has no internal notion of type safety to ensure the referenced tag is only a CoMID or CoSWID, so technically any tag could resolve without any changes to the CDDL.

Reply:
@deeglaze I am not talking about `tag-identity-map` here, I am referring to `concise-tag-type-choice`, which is CoMID/CoSWID/CoTL at the moment, can be extended to include other types